rest_framework/authtoken/serializers.py

```diff
@@ -13,21 +13,17 @@ class AuthTokenSerializer(serializers.Serializer):
     )
 
     def validate(self, attrs):
-        username = attrs.get('username')
-        password = attrs.get('password')
+        username = attrs['username']
```

Comment:
@LondonAppDev 
username & password are CharField, so in order to remove `get() `property you may require to add `allow_blank=False, allow_null=False, required=True` in it just to make sure that these field cant be set null/blank and avoid any Nonetype exception.